prime-router/metadata/fhir_mapping/hl7/datatypes/XTN/ContactPoint.yml

```diff
@@ -0,0 +1,310 @@
+value_1:
+  condition: $telEquipType NOT_EQUALS Internet && $telEquipType NOT_EQUALS X.400 && $localNum NULL && $unformattedTelNum NULL && $telNum NOT_NULL
+  valueOf: $telNum
+  vars:
+    telNum: String, XTN.1
+    telEquipType: String, XTN.3
+    localNum: String, XTN.7
+    unformattedTelNum: String, XTN.12
+
+value_2:
+  condition: $telEquipType NULL && $localNum NULL && $unformattedTelNum NULL && $telNum NOT_NULL
```

Comment:
Again testing if there is a value in the input field so that the element is not created with a blank value. Don't think this is strictly necessary though since it comes from a variable but don't think it hurts either